packages/amazon-ecs-cni-plugins/amazon-ecs-cni-plugins.spec

```diff
@@ -0,0 +1,147 @@
+%global ecscni_goproject github.com/aws
+%global ecscni_gorepo amazon-ecs-cni-plugins
+%global ecscni_goimport %{ecscni_goproject}/%{ecscni_gorepo}
+%global ecscni_gitrev 53a8481891251e66e35847554d52a13fc7c4fd03
+
+Name: %{_cross_os}amazon-ecs-cni-plugins
+Version: %{ecscni_gitrev}
+Release: 1%{?dist}
+Summary: Networking plugins for ECS task networking
+License: Apache-2.0
+URL: https://%{ecscni_goimport}
+Source0: https://%{ecscni_goimport}/archive/%{ecscni_gitrev}/%{ecscni_gorepo}.tar.gz
+
+# Bottlerocket-specific - filesystem location for ECS CNI plugins
+Patch0001: 0001-bottlerocket-default-filesystem-locations.patch
+
+BuildRequires: %{_cross_os}glibc-devel
+
+Requires: %{name}(binaries)
+Requires: (%{name}(ecs-agent-symlinks) if %{_cross_os}variant-family(aws-ecs))
```

Comment:
> thoughts on the subpackages names?

I would probably spell it out as `ecs-agent-extras` and `ecs-agent-fips-extras` since `ext` can be ambiguous (could mean "extension" or "external").